package NKikimrProto;
option java_package = "ru.yandex.kikimr.proto";

message TTypeInfo {
    optional uint32 PgTypeId = 1;
    optional string PgTypeMod = 2;
    optional uint32 DecimalPrecision = 3;
    optional uint32 DecimalScale = 4;
}
